src/main/java/io/github/thebusybiscuit/exoticgarden/listeners/PlantsListener.java

```diff
@@ -353,11 +353,41 @@ public void onDecay(LeavesDecayEvent e) {
         }
     }
 
+    Material[] blockedMaterials = new Material[] {
+    Material.ANVIL,
+    Material.CHIPPED_ANVIL,
+    Material.DAMAGED_ANVIL,
+    Material.DRAGON_EGG,
+    Material.SAND,
+    Material.RED_SAND,
+    Material.GRAVEL,
+    Material.SCAFFOLDING,
+    Material.WHITE_CONCRETE_POWDER,
+    Material.ORANGE_CONCRETE_POWDER,
+    Material.MAGENTA_CONCRETE_POWDER,
+    Material.LIGHT_BLUE_CONCRETE_POWDER,
+    Material.YELLOW_CONCRETE_POWDER,
+    Material.LIME_CONCRETE_POWDER,
+    Material.PINK_CONCRETE_POWDER,
+    Material.GRAY_CONCRETE_POWDER,
+    Material.LIGHT_GRAY_CONCRETE_POWDER,
+    Material.CYAN_CONCRETE_POWDER,
+    Material.PURPLE_CONCRETE_POWDER,
+    Material.BLUE_CONCRETE_POWDER,
+    Material.BROWN_CONCRETE_POWDER,
+    Material.GREEN_CONCRETE_POWDER,
+    Material.RED_CONCRETE_POWDER,
+    Material.BLACK_CONCRETE_POWDER,
+    };
```

Comment:
This is what Tags are for.
I believe Slimefun even has one built-in too.

https://github.com/Slimefun/Slimefun4/blob/master/src/main/java/io/github/thebusybiscuit/slimefun4/utils/tags/SlimefunTag.java#L254

Reply:
Fixed!